dist/dotty.rb

```diff
@@ -0,0 +1,32 @@
+class Dotty < Formula
+  desc "Experimental Scala Compiler"
+  homepage "http://dotty.epfl.ch/"
+  url "http://dotty.epfl.ch/dotty/dotty-0.1.1-bin-SNAPSHOT.tar.gz"
```

Comment:
Will the artifacts always be hosted on dotty.epfl.ch? Do you have scripts to automate the deployment?

I recommend against serving the files from an EPFL server. There have examples where servers hosting artifacts have been compromised http://www.techradar.com/news/popular-video-encoding-mac-app-handbrake-compromised-with-malware I think it would be best to serve these from for example Github releases or Maven Central. I have used https://github.com/aktau/github-release to automate this, but there are other ways.

Additionally, I suspect many will write installation scripts against this url scheme. For example nix, https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/tools/scalafmt/default.nix I would recommend to try and keep this as stable as possible.